### ISSUE TITLE
Weekly Report

### ISSUE THREAD

**Issue by jrspragus** [al-caughey/Google-Meet-Attendance#95]
Is there a way that I could run a weekly report where the attendance shows for each kid for a set period of time ( a few days or a week)? This would be amazing for more ease when seeing patterns of participation. Thanks!

**Comment by Eagle-Man** [comment]
Yes.  In the HTML file, there's a button for Attendance Summary.  Check out the videos on Al's YouTube channel for how to use this.  [This one in particular.](https://youtu.be/91016x4w-i8)
